src/libs/intents/localMethods.ts

```diff
@@ -15,21 +15,27 @@ import { setFlagshipUI } from './setFlagshipUI'
 import { showInAppBrowser, closeInAppBrowser } from './InAppBrowser'
 import { toggleSetting } from '/libs/intents/toggleSetting'
 import { isBiometryDenied } from '/libs/intents/setBiometryState'
-import AsyncStorage from '@react-native-async-storage/async-storage'
+import { clearData } from '/libs/localStore/storage'
+import { clearCookies } from '/libs/httpserver/httpCookieManager'
 
-export const asyncLogout = async (): Promise<null> => {
+export const asyncLogout = async (client?: CozyClient): Promise<null> => {
```

Comment:
As for the test, I think this function should have its own file (like all cozy-intents functions anyway). Not saying we should refactor everything asap, but doing it when adding/updating code